doc/arch/014-region-level-sync.md

```diff
@@ -0,0 +1,47 @@
+# Region level sync
+September 2020 (accepted)
+
+_This ADR has been added retroactively in Apr 2021 to capture our switch to block-level syncing._
```

Comment:
This is essentially the status section. Perhaps we can move that up here and coalesce with this part.